notebooks/official/workbench/ads_targetting/training-multi-class-classification-model-for-ads-targeting-usecase.ipynb

```diff
@@ -1,13 +1,71 @@
 {
```

Comment:
<p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p><p> </p><p><strong>Combine the 2 text cells</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>